src/Model/Photo.php

```diff
@@ -198,12 +201,17 @@ public static function getImageDataForPhoto(array $photo)
 			// legacy data storage in "data" column
 			$i = self::selectFirst(['data'], ['id' => $photo['id']]);
 			if ($i === false) {
-				return null;
+				return '';
```

Comment:
Why the change of return value type?

Reply:
I think it's out of scope as well, but the callers are using the return value either as object .. or as string .. see:
https://github.com/friendica/friendica/blob/c6b929d6d932f1881e0697356efa702f00e2981b/src/Model/Photo.php#L230-L235
I'll revert it, because I'm not sure if there're side effects ..